docs/release-notes/1.6.0.md

```diff
@@ -0,0 +1,61 @@
+# 1.6.0
+
+## General
+
+### Application Instance Log Viewing
+
+Application instance logs can now be viewed directly from the nmaas Portal, enabling easier debugging.
+
+Users can simply navigate to the application instance details page and select the `View Logs` option from the Actions dropdown. Log viewing can be enabled on a per application basis. If a particular instance does not currently expose its logs, contact your nmaas Platform administrator for more information. 
+
+Logs can either be viewed using the dedicated web viewer or downloaded for later analysis. In case a given application instance is comprised of multiple pods, logs are viewable for all of them.
+
+Global administrators can enable log viewing for a given application by ticking the `Allow log access` checkbox when editing a particular version of the application. Note that the log viewing feature can be toggled on a per-version basis.
+
+<figure markdown>
+  ![View Logs Dropdown](img/01-view-logs.png)
+  <figcaption>Accessing the log viewer</figcaption>
+</figure>
+
+<figure markdown>
+  ![Log Viewer](img/02-log-viewer.png)
+  <figcaption>The application instance log viewer</figcaption>
+</figure>
+
+### Deployment Parameters Viewing 
+
+It is now possible to look at all of the parameters used for instantiating a given application instance once it has been deployed. In this release this is possible only for applications with multiple access methods defined, using the `Actions -> Access option`. In a subsequent patch release this feature will become globally available for all application instances.
+
+<figure markdown>
+  ![Application Instance Parameters](img/03-parameter-viewing.png)
+  <figcaption>Viewing deployment parameters of an application instance</figcaption>
+</figure>
+
+### Manual Application Instance Update
+
+Global administrators can now notify the nmaas Platform that an application instance has been manually upgraded in the background, bypassing the built-in update functionality of nmaas. This is useful if a more complex update operation needs to be done, for example between different incompatible major versions, and the administrator has to make manual changes from the CLI.
+
+Once the global administrator has indicated that a manual update has been performed to a new version, the nmaas platform will continue tracking subsequent update paths from that point forward.
+
+The manual update functionality is accessible via the `Actions -> Manual version update` menu of a particular application instance.
+
+<figure markdown>
+  ![Manual Version Update](img/04-manual-version-update.png)
+  <figcaption>Manually updating the version of an application instance</figcaption>
+</figure>
+
+### Automatic Helm Repository Updates in the Background
```

Comment:
This helm repo update execution improvement was actually released in 1.5.4 so I suppose we can add a dedicated page for 1.5.4 as well.